app/commands/solution/search_community_solutions.rb

```diff
@@ -10,11 +10,18 @@ def self.default_per
       DEFAULT_PER
     end
 
-    def initialize(exercise, page: nil, per: nil, criteria: nil)
+    def initialize(exercise, page: nil, per: nil, order: nil,
+                   criteria: nil, tests_status: nil, head_tests_status: nil,
+                   mentoring_status: nil, sync_status: nil)
       @exercise = exercise
       @page = page.present? && page.to_i.positive? ? page.to_i : DEFAULT_PAGE # rubocop:disable Style/ConditionalAssignment
       @per = per.present? && per.to_i.positive? ? per.to_i : self.class.default_per # rubocop:disable Style/ConditionalAssignment
+      @order = order
       @criteria = criteria
+      @tests_status = tests_status
+      @head_tests_status = head_tests_status
+      @mentoring_status = mentoring_status
+      @sync_status = sync_status
```

Comment:
I suggest symbolizing these here, so you know they're symbols elsewhere and can remove the other `to_sym` throughout the command.
```suggestion
      @sync_status = sync_status&.to_sym
```